lib/talktome/app.rb

```diff
@@ -27,7 +27,8 @@ class App < Sinatra::Application
 
     post %r{/([a-z-]+)/} do |action|
       begin
-        TALKTOME.talktome(action, {}, info, [:email]){|email|
+        as_array = info.map{|k,v| {'key' => k.capitalize, 'value' => v}}
+        TALKTOME.talktome(action, {}, {'info' => as_array}, [:email]){|email|
```

Comment:
For backward compatibility (and to better support the usual case), do this instead:

```
info.merge(allvars: as_array)
```

Then use allvars in the email template.